tests/utils.py

```diff
@@ -227,8 +227,8 @@ def get_image_classifier_tf_v1(from_logits=False, load_init=True, sess=None):
     # pylint: disable=E0401
     import tensorflow as tf
 
-    tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.ERROR)
```

Comment:
Why are you moving this line?

Reply:
I was getting errors importing the `tf.compat` module on TF 1. Will remove this until I have working test for tensorflow

Reply:
Actually, you are right, this fixes a bug. Could you please add the change again?